modules/categoryTranslation.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * This module translates iab category to freewheel industry using translation mapping file
+ * Publisher can set translation file by using setConfig method
+ *
+ * Example:
+ * config.setConfig({
+ *    'brandCategoryTranslation': {
+ *      'translationFile': 'http://sample.com'
+ *    }
+ * });
+ * If publisher has not defined translation file than prebid will use default prebid translation file provided here <TODO add url once it is uploaded on cdn>
+ */
+
+import { config } from '../src/config';
+import { hooks, hook } from '../src/hook';
+import { ajax } from '../src/ajax';
+import { timestamp, logError, setDataInLocalStorage, getDataFromLocalStorage } from '../src/utils';
+
+// TODO udpate url once it is uploaded on cdn
+const DEFAULT_TRANSLATION_FILE_URL = 'http://acdn.adnxs.com/prebid/test/jp/freewheel-mapping.json';
```

Comment:
we shouldn't hardcode a protocol here. Use a protocol-less URL to support HTTPS.

This file can also be minified.